### PR TITLE
Add `_all` field for all the indexed lucene documents.

### PR DESCRIPTION
**Pull request by mansu**
Add an `_all` field to all indexed lucene documents so we can perform full text search on the documents.

### CODE REVIEW

**Review by bryanlb — CHANGES_REQUESTED** [review]
Let's make this a config option, but otherwise it looks great!

**Comment by mansu** [comment]
> Let's make this a config option, but otherwise it looks great!

Done!

**Comment by mansu** [comment]
> Changes look good except we still need to set the master `config.yaml` to include this, defaulted to FALSE, and allow an override by ENV similar to the other configs.
> 
> https://github.com/slackhq/kaldb/blob/master/config/config.yaml

Good call out! Updated config.yaml.

File: config/config.yaml [hunk 1]
```diff
@@ -6,6 +6,7 @@ indexerConfig:
   luceneConfig:
     commitDurationSecs: ${INDEXER_COMMIT_DURATION_SECS:-10}
     refreshDurationSecs: ${INDEXER_REFRESH_DURATION_SECS:-11}
+    enableFullTextSearch: ${INDEXER_ENABLE_FULL_TEXT_SEARCH:false}
```
Comment (bryanlb) [inline config/config.yaml:9]: This still isn't quite right - this needs to be `:-false`

**Review by bryanlb — CHANGES_REQUESTED** [review]

File: config/config.yaml (hunk 1 above)
Comment (mansu) [reply config/config.yaml]: oops. Done.

**Review by bryanlb — APPROVED** [review]

**Pull request merged**